src/main/kotlin/io/ionic/libs/ionfileviewerlib/helpers/IONFLVWInputsValidator.kt

```diff
@@ -0,0 +1,33 @@
+package io.ionic.libs.ionfileviewerlib.helpers
+
+import android.os.Looper
+import java.util.regex.Pattern
+
+class IONFLVWInputsValidator {
+    /**
+     * boolean method to check if a given file path is valid
+     * @param path - the file path check
+     * @return true if path is valid, false otherwise
+     */
+    fun isPathValid(path: String?): Boolean {
+        return !path.isNullOrBlank()
+    }
+
+    /**
+     * boolean method to check if a given url is valid
+     * @param url - the url to check
+     * @return true if url is valid, false otherwise
+     */
+    fun isURLValid(url: String): Boolean {
+        val pattern =
+            Pattern.compile("http[s]?://(([^/:.[:space:]]+(.[^/:.[:space:]]+)*)|([0-9](.[0-9]{3})))(:[0-9]+)?((/[^?#[:space:]]+)([^#[:space:]]+)?(#.+)?)?")
+        val matcher = pattern.matcher(url)
+        return matcher.find()
```

Comment:
Can we avoid creating `matcher` and do the following instead?

`return pattern.matcher(url).find()`